core-web/libs/portlets/dot-ema/src/lib/feature/dot-ema.component.ts

```diff
@@ -79,6 +115,38 @@ export class DotEmaComponent implements OnInit {
         });
     }
 
+    ngAfterViewInit(): void {
+        fromEvent(this.window, 'message')
+            .pipe(takeUntil(this.destroy$))
+            .subscribe((event: MessageEvent) => {
+                this.handlePostMessage(event)();
+            });
+    }
```

Comment:
Shouldn't window be available on `ngOnInit`?

Reply:
It is, changed!